src/main/java/org/jitsi/impl/protocol/xmpp/OpSetSubscriptionImpl.java

```diff
@@ -311,10 +311,9 @@ public void handlePublishedItems(ItemPublishEvent event)
          */
         public Subscription(String node, SubscriptionListener listener)
         {
-            Assert.notNull(node, "node");
-            Assert.notNull(listener, "listener");
+            Objects.requireNonNull(listener, "listener");
+            this.node = Objects.requireNonNull(node, "node");
 
-            this.node = node;
             this.listeners.add(listener);
```

Comment:
Will this work here this.listeners.add(Objects.requireNotNull(listener, "listener") ?